app/services/auth/user_wrapper.rb

```diff
@@ -24,5 +24,9 @@ def email
     def token
       auth_info.token
     end
+
+    def actual_user
```

Comment:
maybe call it `user_model` I guess it is more clear what it is

Reply:
We should rethink this user wrapper because `current_user` doesn't really represent a user, and it's quite often seems confusing